styles/globals.css

```diff
@@ -6,6 +6,8 @@
     --font-mono: ui-monospace, Menlo, Monaco, 'Cascadia Mono', 'Segoe UI Mono',
         'Roboto Mono', 'Oxygen Mono', 'Ubuntu Monospace', 'Source Code Pro',
         'Fira Mono', 'Droid Sans Mono', 'Courier New', monospace;
+    --off-brand-dark: #080808;
+    --off-brand-white: #ffff;
```

Comment:
<img width="296" alt="Screenshot 2023-06-01 at 12 06 20 PM" src="https://github.com/ezpzcoders-llc/allie-b-creations/assets/20602890/665906f4-9849-4d34-bf8f-c9aecaa13285">
Already have variables for offwhite and black. use mine for consistency